app/views/parks/show.html.erb

```diff
@@ -39,16 +39,21 @@
 
 <ul>
   <% @park.reviews.each do |review|%>
-    <div class="inline-list">
-      <li>
-        <%= review.title%>
```

Comment:
`<li>` should be the immediate child element of `<ul>`, not `<div>`, and in this case that outer `<div>` seems redundant - you can probably just use `<li class="inline-list">` instead